src/components/backend-ai-data-view.ts

```diff
@@ -431,7 +439,7 @@ export default class BackendAIData extends BackendAIPage {
             </mwc-select>
             ${this.is_admin && this.allowed_folder_type.includes('group') ? html`
               <mwc-select class="fixed-position" id="add-folder-group" ?disabled=${this.folderType==='user'} label="${_t('data.Project')}" FixedMenuPosition>
-                ${(this.allowedGroups as any).map((item, idx) => html`
+                ${(this.allowedGroups as groupData[]).map((item, idx) => html`
```

Comment:
After setting type of property, we can remove `as ...`:
```typescript
@property({type: Array}) allowedGroups:GroupData[] = [];
```

```suggestion
                ${this.allowedGroups.map((item, idx) => html`
```

Reply:
resolved